definitions/oi-kamon-agent/definition.yml

```diff
@@ -0,0 +1,18 @@
+domain: EXT
+type: SERVICE
```

Comment:
@cianBuckley or @srvaroa: For compatibility reasons, I've decided to change these back to just `EXT.SERVICE` instead of `EXT.KAMON`, `EXT.OPENTELEMETRY` et al. Will this cause issues?

Reply:
@srvaroa ping again on this. I am running into validation errors right now, but I wanted to check with you to see if it's ok if we stay with `ext-service` for Open Instrumentation entities in the short-term.

Reply:
Hi, sorry I missed this one.  The relative path seems to be the problem yes, I noticed @jonatns noted this below